tests/test_wave_01.py

```diff
@@ -119,12 +119,17 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    # comment this out after writing asssert statemement
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    #*******************************************************************************************
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
+    assert updated_data["watched"][0]["genre"] == GENRE_1
+    assert updated_data["watched"][0]["rating"] == RATING_1
```

Comment:
👍🏻 Great assert statements! Love that you used all the constants provided. Makes it a very thorough test.